libs/sdk/skyux-eslint/src/configs/ts-strict-base-type-checked.ts

```diff
@@ -24,5 +24,8 @@ export default {
         considerDefaultExhaustiveForUnions: true,
       },
     ],
+    // Using Angular validators (e.g. `Validators.required`) throws this error.
+    // See: https://stackoverflow.com/a/68652060/6178885
+    '@typescript-eslint/unbound-method': ['error', { ignoreStatic: true }],
```

Comment:
This setting was used in the old `@skyux-sdk/eslint-config` package, but it wasn't brought over.